### PR TITLE
Refactoring and Formatting

### PR DESCRIPTION
**Pull request by mrrajan**
Signed-off-by: Rajan Ravi <rravi@redhat.com>

This PR includes,

-  Refactoring and Formatting of Existing DBaaS Test suites
- Updating Test tags
[log.zip](https://github.com/RHODA-lab/rhoda-ci/files/10216911/log.zip)

### CODE REVIEW

File: robotidy.toml [hunk 1]
```diff
@@ -1,16 +1,17 @@
 #########################################################
-# Robotidy configuration for ods-ci coding style guide  #
+# Robotidy configuration for rhoda-ci coding style guide  #
```
Comment (ikanias) [inline robotidy.toml:2]: This is a tool for syntax verification?

File: tests/100_installation.robot [hunk 2]
```diff
@@ -6,29 +6,30 @@ Library             SeleniumLibrary
 Resource            ../resources/keywords/import_provider_account.resource
 
 Suite Setup         Run Keywords
-...                 Set Library Search Order    SeleniumLibrary
-...                 AND    Skip If    ${DBaaSPolicyEnabled}
+...                     Set Library Search Order    SeleniumLibrary
+...                     AND    Skip If    ${DBaaSPolicyEnabled}
```
Comment (ikanias) [inline tests/100_installation.robot:10]: What is this If for?

File: tests/200_dbaas_policy.robot [hunk 3]
```diff
@@ -8,200 +8,202 @@ Resource            ../resources/keywords/dbaas_policy.resource
...
     Then User Is Able To Import RDS Provider Account For ${DBaaSTestNS}
 
 Scenario: Create DBaaSPolicy As Service Admin From Error screen on Admin View And Import RDS
-    [Tags]  dbaas-test-15
+    [Tags]    dbaas-test-15
     Given The User Logs In To Openshift As OCP_LDAP_SERVICE_ADM
     When User Creates DBaaSPolicy For ${DBaaSTestNS} From Invalid Namespace Alert on Admin View As OCP_SERVICE_PROJECT_ADM
     Then User Is Able To Import RDS Provider Account For ${DBaaSTestNS}
 
 Scenario: Create DBaaSPolicy As Project Admin From Admin view And Import MongoDB
```
Comment (ikanias) [inline tests/200_dbaas_policy.robot:109]: Please change the end to: 'Import MongoDB provider account' here and in the rest of the tests.
For consistency and for better understanding :-)

File: tests/300_mongodb_import_connect.robot [hunk 4]
```diff
@@ -8,44 +8,44 @@ Suite Setup         Set Library Search Order    SeleniumLibrary
...
-    [Tags]    smoke    RHOD-49-1
+    [Tags]    RHODA-007    smoke
     When User Filters Project ${operatorNamespace} On Project DropDown And Navigates To Database Access Page
     And User Navigates To Import Database Provider Account Screen From Database Access Page
     And User Enters Invalid Data To Import MongoDB Database Provider Account
     Then Provider Account Import Failure
 
 Scenario: Import MongoDB Provider Account From Administrator View
-    [Tags]    smoke    RHOD-46
+    [Tags]    RHODA-008
```
Comment (ikanias) [inline tests/300_mongodb_import_connect.robot:31]: Shouldn't this also be a smoke test?

File: tests/300_mongodb_import_connect.robot [hunk 5]
```diff
@@ -8,44 +8,44 @@ Suite Setup         Set Library Search Order    SeleniumLibrary
...
     When User Filters Project ${operatorNamespace} On Project DropDown And Navigates To Database Access Page
     And User Navigates To Import Database Provider Account Screen From Database Access Page
     And User Enters Data To Import MongoDB Provider Account
     Then Provider Account Import Success
 
 Scenario: Deploy MongoDB DBSC For MongoDB Provider Account
-    [Tags]    smoke    RHOD-50
+    [Tags]    RHODA-009
```
Comment (ikanias) [inline tests/300_mongodb_import_connect.robot:38]: Shouldn't this also be a smoke test?

File: robotidy.toml (hunk 1 above)
Comment (mrrajan) [reply robotidy.toml:2]: Yes @ikanias

File: tests/100_installation.robot (hunk 2 above)
Comment (mrrajan) [reply tests/100_installation.robot:10]: @ikanias The scope of this PR is to align the existing with the spec on robotidy.yaml.
But, I could recall this "if" statement is skip the execution of entire test suite if the ${DBaaSPolicyEnabled} is set to "True".

File: tests/200_dbaas_policy.robot (hunk 3 above)
Comment (mrrajan) [reply tests/200_dbaas_policy.robot:109]: @ikanias  IMHO, the scope of this PR is for formatting and I dont want to introduce any changes out of the scope. So, Shall I accommodate these suggestion on the next one?

File: tests/300_mongodb_import_connect.robot (hunk 4 above)
Comment (mrrajan) [reply tests/300_mongodb_import_connect.robot:31]: @ikanias This flow would be covered on "Scenario: Connect MongoDB DBSC With An Openshift Application" script execution on the same test suite. So I skipped this one.

File: tests/300_mongodb_import_connect.robot (hunk 5 above)
Comment (mrrajan) [reply tests/300_mongodb_import_connect.robot:38]: @ikanias same as above one.

File: tests/303_mongodb_import_connect_cli.robot [hunk 6]
```diff
@@ -5,29 +5,30 @@ Metadata            Version    0.0.1
...
 Scenario: Verify Error Message For Invalid Credentials On MongoDB Using OC CLI
-    [Tags]    smoke    RHOD-470
+    [Tags]    RHODA-015    smoke
     When User Creates MongoDB Secret With Invalid Credentials
     And User Imports MongoDB Provider Account Using CLI
     Then Provider Account Import Failure Using CLI
 
 Scenario: Import MongoDB Provider Account Using OC CLI
-    [Tags]    smoke    RHOD-460
+    [Tags]    RHODA-016
```
Comment (ikanias) [inline tests/303_mongodb_import_connect_cli.robot:25]: Why isn't this smoke too?

File: tests/400_crunchydb_import_connect.robot [hunk 7]
```diff
@@ -9,41 +9,42 @@ Suite Setup         Set Library Search Order    SeleniumLibrary
...
-    [Tags]    smoke    RHOD-49-2
+    [Tags]    RHODA-021    smoke
     When User Filters Project ${operatorNamespace} On Project DropDown And Navigates To Database Access Page
     And User Navigates To Import Database Provider Account Screen From Database Access Page
     And User Enters Invalid Data To Import CrunchyDB Provider Account
     Then Provider Account Import Failure
 
 Scenario: Import CrunchyDB Provider Account From Administrator View
-    [Tags]    smoke    RHOD-47
+    [Tags]    RHODA-022
```
Comment (ikanias) [inline tests/400_crunchydb_import_connect.robot:32]: smoke?

File: tests/400_crunchydb_import_connect.robot [hunk 8]
```diff
@@ -9,41 +9,42 @@ Suite Setup         Set Library Search Order    SeleniumLibrary
...
     When User Filters Project ${operatorNamespace} On Project DropDown And Navigates To Database Access Page
     And User Navigates To Import Database Provider Account Screen From Database Access Page
     And User Enters Data To Import CrunchyDB Provider Account
     Then Provider Account Import Success
 
 Scenario: Deploy CrunchyDB Database Instance
-    [Tags]    smoke    RHOD-51
+    [Tags]    RHODA-023
```
Comment (ikanias) [inline tests/400_crunchydb_import_connect.robot:39]: smoke?

File: tests/303_mongodb_import_connect_cli.robot (hunk 6 above)
Comment (mrrajan) [reply tests/303_mongodb_import_connect_cli.robot:25]: The flow is covered as part of "Scenario: Deploy MongoDB Provider Account Using OC CLI"

File: tests/400_crunchydb_import_connect.robot (hunk 7 above)
Comment (mrrajan) [reply tests/400_crunchydb_import_connect.robot:32]: Same as above

File: tests/400_crunchydb_import_connect.robot (hunk 8 above)
Comment (mrrajan) [reply tests/400_crunchydb_import_connect.robot:39]: Same as above

File: tests/403_crunchydb_import_connect_cli.robot [hunk 9]
```diff
@@ -5,29 +5,30 @@ Metadata            Version    0.0.1
...
 Scenario: Verify Error Message For Invalid Credentials On CrunchyDB Using OC CLI
-    [Tags]    smoke    RHOD-471
+    [Tags]    RHODA-029    smoke
     When User Creates CrunchyDB Secret With Invalid Credentials
     And User Imports CrunchyDB Provider Account Using CLI
     Then Provider Account Import Failure Using CLI
 
 Scenario: Import CrunchyDB Provider Account Using OC CLI
-    [Tags]    smoke    RHOD-461
+    [Tags]    RHODA-030
```
Comment (ikanias) [inline tests/403_crunchydb_import_connect_cli.robot:25]: smoke?

File: tests/500_cockroachdb_import_connect.robot [hunk 10]
```diff
@@ -9,41 +9,42 @@ Suite Setup         Set Library Search Order    SeleniumLibrary
...
-    [Tags]    smoke
+    [Tags]    RHODA-035    smoke
     When User Filters Project ${operatorNamespace} On Project DropDown And Navigates To Database Access Page
     And User Navigates To Import Database Provider Account Screen From Database Access Page
     And User Enters Invalid Data To Import CockroachDB Provider Account
     Then Provider Account Import Failure
 
 Scenario: Import Cockroach Provider Account From Administrator View
-    [Tags]    smoke
+    [Tags]    RHODA-036
```
Comment (ikanias) [inline tests/500_cockroachdb_import_connect.robot:32]: smoke?

File: tests/500_cockroachdb_import_connect.robot [hunk 11]
```diff
@@ -9,41 +9,42 @@ Suite Setup         Set Library Search Order    SeleniumLibrary
...
     When User Filters Project ${operatorNamespace} On Project DropDown And Navigates To Database Access Page
     And User Navigates To Import Database Provider Account Screen From Database Access Page
     And User Enters Data To Import CockroachDB Provider Account
     Then Provider Account Import Success
 
 Scenario: Deploy CockroachDB Database Instance
-    [Tags]    smoke    RHOD-60
+    [Tags]    RHODA-037
```
Comment (ikanias) [inline tests/500_cockroachdb_import_connect.robot:39]: smoke?

File: tests/403_crunchydb_import_connect_cli.robot (hunk 9 above)
Comment (mrrajan) [reply tests/403_crunchydb_import_connect_cli.robot:25]: Same as above

File: tests/500_cockroachdb_import_connect.robot (hunk 11 above)
Comment (mrrajan) [reply tests/500_cockroachdb_import_connect.robot:39]: Same as above

File: tests/600_rds_import_connect.robot [hunk 12]
```diff
@@ -8,29 +8,29 @@ Resource            ../resources/keywords/deploy_application.resource
...
-    [Tags]    smoke     RHOD-302
+    [Tags]    RHODA-048    smoke
     When User Filters Project ${operatorNamespace} On Project DropDown And Navigates To Database Access Page
     And User Navigates To Import Database Provider Account Screen From Database Access Page
     And User Enters Invalid Data To Import RDS Provider Account
     Then Provider Account Import Failure
 
 Scenario: Import RDS Provider Account From Administrator View
-    [Tags]    smoke    RHOD-300
+    [Tags]    RHODA-049
```
Comment (ikanias) [inline tests/600_rds_import_connect.robot:24]: smoke?

File: tests/500_cockroachdb_import_connect.robot (hunk 10 above)
Comment (mrrajan) [reply tests/500_cockroachdb_import_connect.robot:32]: Same as above

File: tests/600_rds_import_connect.robot (hunk 12 above)
Comment (mrrajan) [reply tests/600_rds_import_connect.robot:24]: Same as above

File: tests/603_rds_import_connect_cli.robot [hunk 13]
```diff
@@ -5,28 +5,29 @@ Metadata            Version    0.0.1
...
 Scenario: Verify Error Message For Invalid Credentials On RDS Using OC CLI
-    [Tags]    smoke    RHOD-510
+    [Tags]    RHODA-059    smoke
     When User Creates RDS Secret With Invalid Credentials
     And User Imports RDS Provider Account Using CLI
     Then Provider Account Import Failure Using CLI
 
 Scenario: Import RDS Provider Account Using OC CLI
-    [Tags]    smoke    RHOD-520
+    [Tags]    RHODA-060
```
Comment (ikanias) [inline tests/603_rds_import_connect_cli.robot:24]: smoke?

Comment (mrrajan) [reply tests/603_rds_import_connect_cli.robot:24]: Same as above

File: robotidy.toml (hunk 1 above)
Comment (ikanias) [reply robotidy.toml:2]: OK

**Review by ikanias — APPROVED** [review]

**Pull request merged**